devs2025.yml

```
@@ -28,19 +28,172 @@ translations:
+    t: |
+      Хоча респондентам загалом знадобилося в середньому 4 заявки на роботу, щоб знайти свою поточну роботу, це число зменшувалося зі збільшенням років досвіду.
+
+      Тим не менш, невелике збільшення спостерігалося на вищому кінці кривої досвіду, що можна пояснити віковою дискримінацією або застарілими навичками, які ускладнюють пошук роботи.
+
+  - key: technology.workplace.job_finding_methods.takeaway.devs2025
+    t: |
+      Наявність більш особистого зв'язку з компанією під час подання заявки (на відміну від використання дошки оголошень про роботу чи соціальних мереж) пов'язана з вищим доходом.
+
+      Єдиний виняток — це наймання через роботу з відкритим кодом, яка пов'язана з тими самими рівнями доходу, що й більш прямі методи, такі як рекрутери чи рекомендації.
+
+  - key: technology.workplace.remote_work_policy.takeaway.devs2025
+    t: |
+      Нестача віддаленої роботи корелює як із нижчим медіанним доходом, так і з нижчим вдоволенням роботою.
+
+  - key: technology.workplace.work_hours_per_week.takeaway.devs2025
+    t: |
+      Чим більше ви працюєте, тим більше заробляєте, принаймні до певної межі. Індія є найпрацьовитішою країною з медіанними 45 годинами роботи на тиждень, тоді як більшість інших країн зосереджені навколо позначки 40 годин.
+
+  - key: technology.workplace.job_happiness.takeaway.devs2025
+    t: |
+      Розробники загалом досить задоволені своєю роботою.
+
+  - key: technology.workplace.workplace_perks.takeaway.devs2025
+    t: |
+      Всупереч стереотипу асоціального комп'ютерного ботаніка, виявляється, що найбільше ми цінуємо насправді один одного — стосунки з колегами займають перше місце серед того, що розробники найбільше цінують у своєму робочому місці.
+
+  - key: technology.workplace.workplace_difficulties.takeaway.devs2025
+    t: |
+      Зрештою, всім нам потрібно заробляти на життя, і погана компенсація посіла перше місце як головна проблема робочого місця, з якою розробники стикаються сьогодні.
+
+  # technology
+
+  # - key: technology.mobile_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.desktop_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.programming_languages.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  - key: technology.productivity_apps.takeaway.devs2025
+    t: |
+      Коли йдеться про додатки для продуктивності, варто відзначити вражаючий результат **Obsidian**, який посів 6-те місце загалом, попри те, що навіть не був частиною попередньо визначених варіантів питання.
+
+      Деякі інструменти, такі як **Azure DevOps** та **Jira**, схильні до використання у великих компаніях, тоді як **Linear** та **Trello** популярніші серед менших організацій.
+
+      При розгляді виключно віддалених організацій,
+
+  - key: technology.chat_apps.takeaway.devs2025
+    t: |
+      Я настільки ж здивований, як і ви, дізнавшись, що **Google Chat** досі існує, проте він посів 9-те місце виключно завдяки відповідям, написаним від руки.
```

```suggestion
      Я так само здивований, як і ви, дізнавшись, що **Google Chat** досі є, а втім він посів 9-те місце виключно завдяки відповідям, написаним від руки.
```
- https://onlinecorrector.com.ua/%D1%82%D0%B0%D0%BA%D0%B8%D0%B9-%D1%82%D0%B0%D0%BA/
- https://onlinecorrector.com.ua/%D1%96%D1%81%D0%BD%D1%83%D0%B2%D0%B0%D1%82%D0%B8-%D0%B1%D1%83%D1%82%D0%B8/